sources/packages/forms/src/form-definitions/sfaa2024-25.json

```diff
@@ -10003,6 +10007,88 @@
               "id": "en74evs",
               "defaultValue": ""
             },
+            {
+              "label": "HTML",
+              "labelWidth": "",
+              "labelMargin": "",
+              "tag": "p",
+              "className": "alert alert-info fa fa-info-circle",
+              "attrs": [
+                {
+                  "attr": "",
+                  "value": ""
+                }
+              ],
+              "content": "<strong>Based on your response, you may be eligible for the Provincial Tuition Waiver Program and the Learning for Future Grant.</strong>\n<br />This means you could have your tuition and eligible fees paid for, and you could also receive an annual grant of $3,500 to cover additional education-related costs (e.g., textbooks, computers, supplies, etc.).\n<br/>To learn more about these financial supports, please visit the StudentAid BC website or contact the Financial Aid Office at your post-secondary institution.",
+              "refreshOnChange": false,
+              "customClass": "banner-info",
+              "hidden": false,
+              "modalEdit": false,
+              "key": "html43",
+              "tags": [],
+              "properties": {},
+              "conditional": {
+                "show": "true",
+                "when": "youthInCare",
+                "eq": "preferNotToAnswer",
```

Comment:
Just double-checking, the information banner should be displayed when "yes" or "preferNotToAnswer" is selected, right?